maintenance/wikia/addLayoutToPortableInfoboxes.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+/**
+ * Script goes through all provided pages in pages param and adds layout="stacked"
+ *
+ * @param string --pages Json encoded array contains list of pages ids and namespaces
+ * e.g. before encoding
+ * [
+ *  [
+ *    'page_id' => 123,
+ *    'namespace' => 0
+ *  ],
+ *  [
+ *    'page_id' => 124,
+ *    'namespace' => 10
+ *  ]
+ * ]
+ *
+ * @author Kamil Koterba
+ * @ingroup Maintenance
+ */
+
+require_once( dirname( __FILE__ ) . '/../Maintenance.php' );
+
+/**
+ * Maintenance script class
+ */
+class addLayoutToPortableInfoboxes extends Maintenance {
+
+	private $summaries = [
+		'en' => 'Adding explicit layout parameter'
+	];
+
+	public function __construct() {
+		parent::__construct();
+	}
+
+	public function execute() {
+		$pages = json_decode( $this->getOption( 'pages' ) );
+		foreach ( $pages as $page ) {
+			if ( $page['namespace'] !== 10 ) {
```

Comment:
What about the in-place infoboxes (ones directly embedded in an article)? They make up for 2% of all infobox definitions